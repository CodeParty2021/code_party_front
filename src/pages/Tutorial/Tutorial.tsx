import React from "react";
import { Link } from "react-router-dom";
type Props = {};

export const Tutorial: React.FC<Props> = () => {
  return (
    <div>
      <h1>チュートリアルのステップでのコーディングページ</h1>
      <Link to="/tutorial/world/:world_id/mission/:mission_id/end">
        チュートリアルを終了する
      </Link>
    </div>
  );
};
