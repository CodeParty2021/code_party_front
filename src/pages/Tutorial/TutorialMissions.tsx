import React from "react";
import { Link } from "react-router-dom";
type Props = {};

export const TutorialMissions: React.FC<Props> = () => {
  return (
    <div>
      <h1>チュートリアルミッション一覧のページ</h1>
      <Link to="/tutorial/world/:world_id/mission/:mission_id/step/:step_id">
        ミッションXのステップXに進む
      </Link>
      <Link to="/tutorial/world/:world_id/intro">
        このワールドのゲーム説明に進む
      </Link>
    </div>
  );
};
