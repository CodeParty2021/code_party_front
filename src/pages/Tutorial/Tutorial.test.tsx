import React from "react";
import { shallow } from "enzyme";
import { Tutorial } from "./Tutorial";
describe("<Tutorial />", () => {
  it("auth snapshot test", () => {
    const wrapper = shallow(<Tutorial />);

    expect(wrapper.getElements()).toMatchSnapshot();
  });
});
