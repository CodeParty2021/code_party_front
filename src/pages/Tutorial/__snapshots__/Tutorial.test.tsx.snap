// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tutorial /> auth snapshot test 1`] = `
Array [
  <div>
    <h1>
      チュートリアルのステップでのコーディングページ
    </h1>
    <Link
      to="/tutorial/world/:world_id/mission/:mission_id/end"
    >
      チュートリアルを終了する
    </Link>
  </div>,
]
`;
